src/cmd/internal.go

```diff
@@ -166,66 +161,6 @@ tableOfContents: false
 	},
 }
 
-func addGoComments(reflector *jsonschema.Reflector) error {
-	addCommentErr := errors.New("this command must be called from the root of the Zarf repo")
-
-	typePackagePath := filepath.Join("src", "api", "v1alpha1")
-	if err := reflector.AddGoComments("github.com/zarf-dev/zarf", typePackagePath); err != nil {
-		return fmt.Errorf("%w: %w", addCommentErr, err)
-	}
-	varPackagePath := filepath.Join("src", "pkg", "variables")
-	if err := reflector.AddGoComments("github.com/zarf-dev/zarf", varPackagePath); err != nil {
-		return fmt.Errorf("%w: %w", addCommentErr, err)
-	}
-	return nil
-}
-
-var genConfigSchemaCmd = &cobra.Command{
-	Use:     "gen-config-schema",
-	Aliases: []string{"gc"},
-	Short:   lang.CmdInternalConfigSchemaShort,
-	RunE: func(_ *cobra.Command, _ []string) error {
-		reflector := jsonschema.Reflector(jsonschema.Reflector{ExpandedStruct: true})
-		if err := addGoComments(&reflector); err != nil {
-			return err
-		}
-
-		schema := reflector.Reflect(&v1alpha1.ZarfPackage{})
-		output, err := json.MarshalIndent(schema, "", "  ")
-		if err != nil {
-			return fmt.Errorf("unable to generate the Zarf config schema: %w", err)
-		}
-		fmt.Print(string(output) + "\n")
-		return nil
-	},
-}
-
-type zarfTypes struct {
-	DeployedPackage types.DeployedPackage
-	ZarfPackage     v1alpha1.ZarfPackage
-	ZarfState       types.ZarfState
-}
-
-var genTypesSchemaCmd = &cobra.Command{
-	Use:     "gen-types-schema",
```

Comment:
I didn't build a replacement for this as I don't think it's needed. I'm not sure what the original use case was for adding this.